### DISCUSSION TITLE
Unsure on json Format for Padding Configuration

### DISCUSSION THREAD

**Discussion by Medic-2016** [fastfetch-cli/fastfetch#1195] [Q&A]
I'm needing help adding  `--logo-padding-top 3 --logo-padding-left 2` as a permanent config in the json file. I'm unsure on the format for json in general. I tried to add it but it errors out. My config is the standard generated config. Any help would be much appreciated!

**Comment by CarterLi** [comment]
`fastfetch --logo-padding-top 3 --logo-padding-left 2 --gen-config`
